src/PKSim.Presentation/Presenters/Snapshots/LoadSimulationFromSnapshotPresenter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Services;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility.Events;
+using PKSim.Core.Model;
+using PKSim.Core.Services;
+using PKSim.Core.Snapshots.Mappers;
+using PKSim.Core.Snapshots.Services;
+using PKSim.Presentation.Views.Snapshots;
+
+namespace PKSim.Presentation.Presenters.Snapshots
+{
+   public interface ILoadSimulationFromSnapshotPresenter : ILoadFromSnapshotPresenter<Simulation>
+   {
+      /// <summary>
+      ///    Simulation loaded from selected snapshot file. It is null if the user cancels the action or if the file was not
+      ///    loaded properly
+      /// </summary>
+      Simulation LoadSimulation();
+   }
+
+   public class LoadSimulationFromSnapshotPresenter : LoadFromSnapshotPresenter<Simulation>, ILoadSimulationFromSnapshotPresenter
```

Comment:
Also created an overload for Simulation so that we have the same UI even if it's only for debug